e2e/integration/helpers/helpers.go

```diff
@@ -0,0 +1,341 @@
+		utils.Failf("HTTP request is failed :%v", err)
+		return false
+	}
+	return true
+}
+func DeleteDeployment(DeploymentApi, deploymentname string)int{
+	err, resp := SenHttpRequest(http.MethodDelete, DeploymentApi+"/"+deploymentname)
+	if err != nil {
+		// handle error
+		utils.Failf("HTTP request is failed :%v", err)
+		return -1
+	}
+
+	defer resp.Body.Close()
+
+	return resp.StatusCode
+
+}
+//Function to get the pods from Edged
+func GetPods(apiserver, label string) (v1.PodList, error) {
+	var pods v1.PodList
+	var resp *http.Response
+	var err error
+
+	ls := utils.MapLables(map[string]string{"app":label})
+	if len(ls) > 0{
+		err, resp = SenHttpRequest(http.MethodGet, apiserver+podLabelSelector+ls)
+	}else {
+		err, resp = SenHttpRequest(http.MethodGet, apiserver)
+	}
+	if err != nil {
+		utils.Failf("Frame HTTP request failed: %v", err)
+		return pods, nil
+	}
+	defer resp.Body.Close()
+	contents, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		utils.Failf("HTTP Response reading has failed: %v", err)
+		return pods, nil
+	}
+	err = json.Unmarshal(contents, &pods)
+	if err != nil {
+		utils.Failf("Unmarshal HTTP Response has failed: %v", err)
+		return pods, nil
+	}
+	return pods, nil
+}
+
+func GetPodState(apiserver string) (string, int){
+	var pod v1.Pod
+
+	err, resp := SenHttpRequest(http.MethodGet, apiserver)
+	if err != nil {
+		utils.Failf("GetPodState :SenHttpRequest failed: %v", err)
+	}
+	defer resp.Body.Close()
+
+	if resp.StatusCode != http.StatusNotFound{
+		contents, err := ioutil.ReadAll(resp.Body)
+		if err != nil {
+			utils.Failf("HTTP Response reading has failed: %v", err)
+		}
+		err = json.Unmarshal(contents, &pod)
+		if err != nil {
+			utils.Failf("Unmarshal HTTP Response has failed: %v", err)
+		}
+		return string(pod.Status.Phase), resp.StatusCode
+	}
+
+	return " ", resp.StatusCode
+}
+//Function to check the Pod state
+func CheckPodRunningState(apiserver string, podlist v1.PodList) {
+	Eventually(func() int {
+		var count int
+		for _, pod := range podlist.Items {
+			state, _:= GetPodState(apiserver+ "/"+pod.Name)
+			utils.InfoV2("PodName: %s PodStatus: %s", pod.Name, state)
+			if state == "Running"{
+				count ++
+			}
+		}
+		return count
+	}, "120s", "4s").Should(Equal(len(podlist.Items)), "Delete Application deployment is Unsuccessfull, Pod has not come to Running State")
+
+}
+
+//Function to check the Pod state
+func CheckPodDeleteState(apiserver string, podlist v1.PodList) {
+	Eventually(func() int {
+		var count int
+		for _, pod := range podlist.Items {
+			status, statusCode := GetPodState(apiserver+ "/"+pod.Name)
+			utils.InfoV2("PodName: %s status: %s StatusCode: %d", pod.Name, status, statusCode)
+			if statusCode == 404{
+				count ++
+			}
+		}
+		return count
+	}, "120s", "4s").Should(Equal(len(podlist.Items)), "Delete Application deployment is Unsuccessfull, Pod has not come to Running State")
+
+}
+
+func GenerateNodeReqBody(nodeid string) (error, map[string]interface{}){
```

Comment:
Pls add a comment with a short description above this function explaining its functionality